shape-blog-react/config/webpack.config.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+const path = require('path');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const CaseSensitivePathsPlugin = require('case-sensitive-paths-webpack-plugin');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const webpack = require('webpack');
+const devMode = process.env.NODE_ENV !== 'production';
+const packageJson = require('../package.json');
+const autoprefixer = require('autoprefixer');
+
+const hotloader = [
+  'webpack-dev-server/client?http://localhost:3000',
+  'webpack/hot/only-dev-server'
+];
+
+const app = [path.resolve(__dirname, '../static/index.js')];
```

Comment:
In `tsg deploy` the `static` path is zipped up and sent to TakeShape. This means that your uncompiled source files are being deployed. This is why other projects store their files that need to be processed in a `src` directory.